src/apps/opening-hours-editor/OpeningHoursEditor.tsx

```diff
@@ -41,7 +41,7 @@ const OpeningHoursEditor: React.FC = () => {
       }
       eventClick={handleEventClick}
       eventContent={(eventInput) =>
-        renderOpeningHoursEditorEventContent({
+        OpeningHoursEditorEventContent({
```

Comment:
In regards to your commit message, is it something like this you are looking for ? 

```
     eventContent={(eventInput) => {
        return (
          <OpeningHoursEditorEventContent
            eventInput={eventInput}
            handleEventRemove={() =>
              handleEventRemove({ eventToRemove: eventInput, events })
            }
          />
        );
      }}
      
```